### PR TITLE
✨ feat: Added Koin

### PR DESCRIPTION
**Pull request by OdisBy**
## Task

This pull request refers to task LOR-XX

- [x] Does not apply to this pull request

## Description

I added Koin and one example of usage for repository and viewmodel, for more informations see Discord.

## Types of changes #

- [ ] New feature (non-breaking change which adds functionality).
- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected).

## What changed: #

- [ ] UI
- [x] Business rule
- [ ] Documentation
- [ ] Gradle or Build
- [ ] Navigation
- [ ] Tests
- [ ] Resource files
- [x] Project architecture
- [ ] Code style

## Checklist #

- [x] Have tested the changes.
- [ ] Met all the acceptance requirements of this task.
- [x] Verified if branch is up-to-date with `development` (if not - rebase it or merge it).
- [ ] Resolve git conflicts

## If any of the acceptance criteria are not met, please explain why below #
What is the criteria? Why it is different?

### CODE REVIEW

File: feature/auth/register/src/main/java/com/loryblu/feature/auth/register/child/ChildRegisterScreen.kt [hunk 1]
```diff
@@ -243,14 +242,14 @@ fun ChildRegisterScreen(
...
-fun PreviewComposable() {
-    ChildRegisterScreen(
-        viewModel = ChildRegisterViewModel(),
-        navigateToConfirmationScreen = {},
-        onSignUpButtonClicked = {},
-        shouldGoToNextScreen = false,
-        intentForPrivacy = Intent(Intent.ACTION_VIEW),
-    )
-}
+//@Composable
```
Comment (hartmannjean) [inline feature/auth/register/src/main/java/com/loryblu/feature/auth/register/child/ChildRegisterScreen.kt:245]: OK, it will be fix later

**Review by hartmannjean — APPROVED** [review]
👌🏼

**Comment by softdevandre** [comment]
👌🏼

**Review by softdevandre — APPROVED** [review]

File: feature/auth/register/src/main/java/com/loryblu/feature/auth/register/child/ChildRegisterScreen.kt (hunk 1 above)
Comment (OdisBy) [reply feature/auth/register/src/main/java/com/loryblu/feature/auth/register/child/ChildRegisterScreen.kt:245]: For save https://jetc.dev/slack/2021-04-17-preview-viewmodel.html

**Pull request merged**